providers/fips/self_test_kats.c

```diff
@@ -647,6 +647,222 @@ static int self_test_asym_cipher(const ST_KAT_ASYM_CIPHER *t, OSSL_SELF_TEST *st
     return ret;
 }
 
+static const char *self_test_kem_description(const ST_KAT_KEM *t,
+                                             const char *test)
+{
+    static char buf[100];
```

Comment:
If you are going to do this I would prefer the buffer to be passed in

Reply:
This is done now.